feature-libs/cart/base/core/store/actions/cart.action.ts

```diff
@@ -54,13 +54,20 @@ export class CreateCart extends StateUtils.EntityLoadAction {
   }
 }
 
-interface CreateCartFailPayload extends CreateCartPayload {
-  error: ErrorActionType;
-}
-
```

Comment:
[NOTE]

Removed this interface in favor of inline info about `error`:
- `CreateCartPayload & { error: any }`
- `CreateCartPayload & { error: `ActionErrorProperty }`